handbook/handbook.sop.git-workflow.md

```diff
@@ -0,0 +1,86 @@
+---
+id: tIFX0R6iEvZUGkOJluYo1
+title: Git Workflow
+desc: ''
+updated: 1624216920883
+created: 1624215795020
+---
+
+## Summary
+
+How we work with code and git
+
+## Process
+
+We employ the [feature branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) when working with git. 
+
+### Working on a new Feature
+This means that every feature will go into a feature branch during local development and pushed to the origin with the same name. When changes are ready to be reviewed, submit a pull request against `master`. 
+
+For feature branches, we use the following conventions:
+- feat/{feature-name}
+- fix/{fix-name}
+
+### Commit Style
+- we follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) with the following tags
```

Comment:
Are we requiring these for the commits within development branches? I prefer skipping commit naming rules because my work style is sometimes somewhat disorganized, and it takes extra effort to separate all the changes into clean commits I can name properly.

I think that's okay since the individual commits within the pull request are not directly visible in the history as long as the pull request is merged with no-fast-forward (default in Github). If it's useful for reviews however, I am happy to start properly tagging my commits.

Also, should we recommend tagging pull request names in this way as well? The pull request name can be used to tag the merge commit with this style.

Reply:
for naming inside development branches, `spike:` exists as an escape hatch. anything that isn't complete can be prefixed with that

and yes, great suggestion. updated the pr